src/components/NewQuestionnaire/Questions.vue

```diff
@@ -1,27 +1,42 @@
 <template>
-  <QuestionTitle :title="'タイトル'" />
+  <Card>
+    <template #header>
+      <QuestionnaireTitle :title="'タイトル'" />
+    </template>
+  </Card>
   <div>
     <div v-if="questions.length === 0">質問がありません</div>
-    <div v-for="(question, i) in questions" :key="i">
-      <TextForm
-        v-if="isTextForm(question)"
-        :index="i"
-        :question-data="question"
-        @update="updateQuestion"
-      />
-      <ChoiceForm
-        v-if="isChoiceForm(question)"
-        :index="i"
-        :question-data="question"
-        :is-radio="question.question_type === QuestionType.MultipleChoice"
-        @update="updateQuestion"
-      />
-      <LinearScaleForm
-        v-if="isLinearScaleForm(question)"
-        :index="i"
-        :question-data="question"
-        @update="updateQuestion"
-      />
+    <div v-for="(question, i) in questions" :key="question.key">
```

Comment:
```suggestion
    <div v-for="(question, i) in questions" :key="question.key">
```
よくよく考えれば`key`を特別用意しなくていい気がしました
`**Form`のほうで`props`を`ref`でとってるところを`computed`にするといい感じになる気がします(`computed`にすることで強制的に再レンダリングさせなくても`props`に追従する)
以下は`ChoiceForm`の例
```
    const ChoiceQuestions = computed(() => props.questionData.options)
...
    const updateChoice = (label: string, index: number) => {
      const tmp = [...ChoiceQuestions.value]
      tmp[index] = label
      updateChoiceQuestionData('options', tmp)
    }
```